Example/Example/SeriesDetailViewController.swift

```diff
@@ -9,6 +9,92 @@
 import Foundation
 import BothamUI
 
-class SeriesDetailViewController : ExampleViewController {
+private struct Config {
+    static let numberOfColumns = 3
+    static let cellHeight = 174
+    static let headerHeight = 534
+    static let footerHeight = 20
+    static let cellMargin = 20
+}
+class SeriesDetailViewController : ExampleViewController, BothamCollectionViewController, SeriesDetailUI, UICollectionViewDelegateFlowLayout, UICollectionViewDelegate, UICollectionViewDataSource {
```

Comment:
You are at the same time implementing UICollectionViewDelegateFlowLayout, UICollectionViewDelegate,
UICollectionViewDelegateFlowLayout already include UICollectionViewDelegate